sim_interface/source/sim_interface.cpp

```diff
@@ -857,7 +845,7 @@ Sim_output Sim_interface::simulate(const Sim_input& input)
         compute_talent_weight(simulator_talent, character, talents_info, "Improved Overpower", config,
                               &Combat_simulator_config::talents_t::overpower, 2);
 
-        if (config.combat.use_slam && is_two_handed)
+        if (config.combat.use_slam && is_dual_wield)
```

Comment:
Can you elaborate on this and several ``is_two_handed`` that has been replaced with ``is_dual_wield``?
Wouldn't this meant that it'll compute slam talent if you are dual wielding?

Reply:
Good catch. I think this is the fallout of an incomplete refactoring. Looking into it.

Reply:
Ye, I noticed that `is_two_handed` was `!character.is_dual_wield()`, and planned to invert it everywhere (as the first couple of usages where `!is_two_handed`). Obviously I stopped after a few lines, and forgot about it.